docs/quickstart/README.md

```diff
@@ -451,6 +451,7 @@ This output represents the backend response. Now you should be able to make Torn
 ### Step 3b: Connecting to the Tornjak frontend to access the Tornjak UI
 
 Make sure that the backend is accessible from your browser at `http://localhost:10000`, as above, or the frontend will not work.
+- Tip: run kubectl -n spire port-forward spire-server-0 10000:10000 as a background process that way you can access the frontend.
```

Comment:
```
- Tip: Make sure the above command is running in the background or in a separate terminal so that the frontend can access the backend
```

^ The problem is more about the frontend accessing the backend properly than about the user accessing the frontend